Genome/Source/Foundation/Foundation.swift

```diff
@@ -8,6 +8,7 @@
 //  MIT
 
 import PureJsonSerializer
+import Foundation
```

Comment:
I think they changed something to do with the default import of Foundation when dealing with some frameworks.  Probably a good addition going forward.